projects/angular/src/forms/styles/_number-input.clarity.scss

```
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+@use '../../utils/mixins';
+@use 'mixins.forms' as forms-mixins;
+@use 'variables.forms' as forms-variables;
+@use '@cds/core/tokens/tokens.scss' as tokens;
+@use 'sass:selector';
+
+@include mixins.exports('forms.number-input') {
+  .clr-number-input-wrapper {
+    display: inline-block;
+    white-space: nowrap;
+    max-height: forms-variables.$clr-forms-input-wrapper-height;
+
+    .clr-number-input {
+      /* Chrome, Safari, Edge, Opera */
+      &::-webkit-outer-spin-button,
+      &::-webkit-inner-spin-button {
+        -webkit-appearance: none;
+        margin: 0;
+      }
+      /* Firefox */
+      -moz-appearance: textfield;
+
+      &-separator {
+        border: tokens.$cds-alias-object-border-width-100 solid tokens.$cds-alias-object-border-color;
+        width: tokens.$cds-global-space-1;
+        height: tokens.$cds-global-space-7;
+      }
+    }
+
+    .clr-input-group-icon-action {
+      @include mixins.equilateral(calc(tokens.$cds-global-space-9 - tokens.$cds-alias-object-border-width-200));
+      padding: 0 tokens.$cds-global-space-3;
+      &:hover {
+        background-color: tokens.$cds-alias-object-interaction-background-hover;
+      }
+      &:active {
+        background-color: tokens.$cds-alias-object-interaction-background-active;
+      }
+      &:disabled {
+        background-color: unset;
```

`+` and `-` should have some disabled color.

```suggestion
        fill: forms-variables.$clr-forms-text-disabled;
        background-color: unset;
```